.github/workflows/ci.yml

```diff
@@ -36,7 +37,7 @@ jobs:
         run: dotnet build --no-restore
 
       - name: Test
-        run: dotnet test --no-build --verbosity normal -- RunConfiguration.TargetPlatform=x64
```

Comment:
Isn't this removing the Mac x86-64 CI? We still need both

Reply:
as this step is executed for all matrix os's, it will get executed for macos-latest in x64 and for macos-14 in arm64

Reply:
yep thats how I read that, it will execute for both steps and use the systems platform+arch, the latter of which will differ in the case of the two distinct macos images which are arch specific